src/azure-cli/azure/cli/command_modules/acs/_help.py

```diff
@@ -601,6 +612,10 @@
     text: az aks update -g MyResourceGroup -n MyManagedCluster --enable-managed-identity
   - name: Update the cluster to use user assigned managed identity in control plane.
     text: az aks update -g MyResourceGroup -n MyManagedCluster --enable-managed-identity --assign-identity <user_assigned_identity_resource_id>
+  - name: Update a managed AAD AKS cluster to use Azure RBAC
```

Comment:
both managed aad and Azure RBAC can be enabled together, right?

Reply:
this is update command. So can assume managed aad is enabled . Hence the description is Update managed Aad AKS cluster

Reply:
can't customer enabled managed aad and Azure RBAC on non-aad cluster together?

Reply:
they can but this help is specifically for enable-azure-rbac

Reply:
there is a help for enable-aad as well

Reply:
added another help